src/Engines/nativeEngine.ts

```diff
@@ -763,7 +763,7 @@ class CommandBufferEncoder {
 /** @hidden */
 export class NativeEngine extends Engine {
     // This must match the protocol version in NativeEngine.cpp
-    private static readonly PROTOCOL_VERSION = 2;
+    private static readonly PROTOCOL_VERSION = 3;

```

Comment:
Was there a change to the protocol?

Reply:
Yes, I added `offset` and `length` params to several transform functions that have native implementations.